### DISCUSSION TITLE
Kickers don't use crew skill

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#549] [General]
I discovered for sure this morning that kickers don't take advantage of crew skill.  I noticed a long time ago that they seem to be less reliable than the built-in kickers of the Apollo/Soyuz or direct ascent capsules, but this would go a long way toward explaining why: if the TLI stage really is at 94%, rather than 97 or 98% due to a CA3 or CA4 flying the ship, it would fail much more often.  I have a feeling this was intentional and that it's all part of game balance, part of the extra chance you take with Gemini/Voskhod and a drawback for the minishuttles.  Anyone else have an idea on this?

![Kicker](https://user-images.githubusercontent.com/3118014/121783118-c60dee00-cb61-11eb-999a-7fd351e97c01.png)

**Comment by hweimer** [comment]
I'm not sure how much a pilot's skill could affect the outcome of a TLI burn anyway. One could maybe argue that the tight integration with the capsule program offers more options to the pilot in case of trouble. Concerning balance, the kicked Gemini/Voskhod is already borderline powerful (not surprising, since Lunar Gemini would have been both cheaper and faster than Apollo in real life), so removing one of its drawbacks is probably not a good idea.

**Reply by peyre** [reply]
I agree with you.  I think the kicker is considered a sort of rocket component anyway, so it makes sense that crew skill wouldn't apply to it.

**Reply by hweimer** [reply]
Yeah, but then one could argued that this should apply to capsules with built-in kickers as well. However, when you roll against the safety factor of the capsule, you usually have the 'naut skill automatically included, so this would be not so trivial to implement.

**Reply by peyre** [reply]
Right.  I think it's good to leave the handling of kickers alone - seems to me it's a helpful disadvantage for Gemini/Voskhod.  I've asked myself why I even created this discussion item - I think it's just that I finally confirmed to myself that crew skill isn't added to it.  I always knew it was more dangerous to use a discrete kicker than Apollo's built-in, and now I finally understand why.

**Comment by rnyoakum** [comment]
I think the consensus is that kickers are best left alone for game balance, but it strikes me that the best skill to apply to a kicker test would be Docking/Navigation (per [previous discussion](https://github.com/raceintospace/raceintospace/issues/384#issuecomment-671569498)).

**Reply by peyre** [reply]
I actually agree we should leave it alone.  Not applying crew skill is a big part of the drawback to using them, part of balancing Gemini's advantages.  I only started this thread because I was struck suddenly that, of course, as an upper booster stage they wouldn't use piloting skill.

**Comment by zombiedesigner** [comment]
Yes, game balance is critical, there is a reason the Apollo Sat was chosen as LOR.
